ebu_tt_live/node/test/test_handover_unit.py

```diff
@@ -0,0 +1,92 @@
+
+from ebu_tt_live.node import HandoverNode
+from unittest import TestCase
+from mock import MagicMock
+from ebu_tt_live.carriage.interface import IProducerCarriage
+from ebu_tt_live.documents import EBUTT3Document
+
+
+class HandoverUnittests(TestCase):
+
+    _sequence_identifier = 'test_output_sequence'
+    _authors_group_identifier = 'test_group'
+
+    def setUp(self):
+        carriage = MagicMock(spec=IProducerCarriage)
+        carriage.expects.return_value = HandoverNode.provides()
+        self.handover = HandoverNode(
+            node_id='test_handover_node',
+            authors_group_identifier=self._authors_group_identifier,
+            sequence_identifier=self._sequence_identifier,
+            producer_carriage=carriage
+        )
+
+    def _create_test_document(self, sequence_identifier, sequence_number, authors_group_identifier=None,
```

Comment:
If you find it let me know :) Seriously how is this not readable? Nothing is shortened everything has long verbose names...